src/middlewares/authMiddleware.js

```diff
@@ -0,0 +1,24 @@
+import { jwtService } from '../services/jwt.service.js';
+
+export const authMiddleware = (req, res, next) => {
+  const authorization = req.headers['authorization'] || '';
+  const [, token] = authorization.split(' ');
+
+  if (!authorization || !token) {
+    res.sendStatus(401);
+
+    return;
+  }
+
+  const userData = jwtService.verify(token);
```

Comment:
Ensure that jwtService.verify handles exceptions properly. If it throws an error, it should be caught and handled to prevent the server from crashing.